source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -351,6 +351,10 @@ def get(self, region=None, lon=None, lat=None):
         if args['disable_geojson']:
             g.disable_geojson = True
 
+        user = authentication.get_user(token=authentication.get_token(), abort_if_no_token=False)
+
+        args['shape'] = json.loads(user.shape) if user and user.shape else None
```

Comment:
yes I think it's more coherent, the different autocompletes system might not know how to handle all the parameters but it's the same as the coordinate (to boost the results near the coordinate)